docs/installation.rst

```diff
@@ -1,24 +1,76 @@
 Installation
 ============
 
-This module is developed using pipenv facilities. In order to manage virtual
+Via pip
+-------
+
+Since SEM is under active development, the recommended method to get all the
+latest features is using pip to install directly from the develop branch of the
+Github repository:
+
+.. code:: bash
+
+  pip install git+https://github.com/DvdMgr/sem.git@develop
+
+Alternatively, the master branch is more stable but also offers fewer features:
+
+.. code:: bash
+
+  pip install git+https://github.com/DvdMgr/sem.git@master
+
+Installing ns-3
+---------------
+
+An ns-3 installation is required to use SEM. Detailed setup instructions for
+ns-3 are available here_, however, in most cases, cloning the ns-3 project is
+enough to get started. The following command downloads the development version
+of ns-3 to the ns-3-dev folder:
+
+  .. _here: https://www.nsnam.org/wiki/Installation
+
+.. code:: bash
+
+  git clone https://github.com/nsnam/ns-3-dev-git ns-3-dev
```

Comment:
Add comments about how does SEM work for the released version.

Reply:
Done!